enumerables.rb

```diff
@@ -0,0 +1,262 @@
+    return to_enum unless block_given?
+
+    invited_list = []
+    my_each do |friend|
+      invited_list.push(friend) if yield(friend)
+    end
+
+    invited_list
+  end
+
+  # My  All
+  # rubocop:disable Metrics/AbcSize
+
+  def my_all?(args = nil)
+    if block_given?
+      my_each do |element|
+        return false if yield(element) == false
+      end
+    elsif args.nil?
+      my_each do |i|
+        return false if i == false || i.nil?
+      end
+    elsif !args.nil? && (args.is_a? Class)
+      my_each do |i|
+        return false unless i.is_a? args
+      end
+    elsif !args.nil? && args.class == Regexp
+      my_each do |i|
+        return false unless args.match(i)
+      end
+    else
+      my_each do |i|
+        return false if i != args
+      end
+    end
+    true
+  end
+
+  # my_Any
+  def my_any?(args = nil)
+    check = false
+    is_true = false
+    if block_given?
+      my_each do |i|
+        if yield(i) == true
+          check = true
+          return check
+        else
+          check = false
+        end
+      end
+      return check
+    elsif args.class == Regexp
+      my_each do |i|
+        return true if i.match(args)
+
+        return false
+      end
+    elsif !args.nil? && (args.is_a? Class)
+      my_each do |i|
+        if i.is_a? args
+          is_true = true
+          return true
+        else
+          is_true = false
+        end
+      end
+      return is_true
+    elsif !(args.is_a? Class) && args != Regexp
+      if !args.nil?
+        my_each do |i|
+          return true if i == args
+        end
+        return false
+      else
+        mah_chk = false
+        my_each do |i|
+          if i.nil? || i == false
+            mah_chk = false
+          else
+            mah_chk = true
+            return true
+          end
+        end
+        return mah_chk
+      end
+    end
+    unless block_given?
+      my_each { |x| return true if x == true }
+      return false
+    end
+    false
+  end
+
+  def my_none?(args = nil)
+    if block_given?
+      check = false
+      my_each do |element|
+        if yield(element) == false
+          check = true
+        else
+          check = false
+          return check
+        end
+      end
+      return check
+    elsif !args.nil? && (args.is_a? Class)
+      my_each do |i|
+        return false if i.is_a? args
+      end
+    elsif args.class == Regexp
+      my_each do |i|
+        return false if i.match(args)
+
+        return true
+      end
+    elsif !args.nil?
+      chk_array = []
+      param_chk = false
+      my_each do |i|
+        if args != i
+          param_chk = 1
+          chk_array.push(param_chk)
+        else
+          param_chk = 0
+          return false
+        end
+      end
+      return chk_array.my_all?(1)
+    end
+    unless block_given?
+      my_each { |x| return false if x == true }
+      return true
+    end
+    false
+  end
+
+  # My Inject
+  def my_inject(param = nil, sym = nil)
```

Comment:
This should raise `LocalJumpError` if no arguments are given.

Reply:
It is raising the error

Reply:
![Screenshot from 2020-07-08 10-08-30](https://user-images.githubusercontent.com/43162770/86928802-0ae40600-c103-11ea-83a1-0cb18c6461c3.png)

Reply:
According to past reviewer @kobiyoyo as well as actual requirement of inject enum, the error should be "no block given (LocalJumpError)" instead of "LocalJumpError" if no arguments are given.